src/Bridge/Symfony/Routing/ApiLoader.php

```diff
@@ -87,6 +87,9 @@ public function load($data, $type = null): RouteCollection
 
             if (null !== $collectionOperations = $resourceMetadata->getCollectionOperations()) {
                 foreach ($collectionOperations as $operationName => $operation) {
+                    if ('subresource' === substr($operationName, -11)) {
```

Comment:
Either this or we add a `SubresourceOperations` in the resource metadata declarations. Lmk.

Reply:
I think SubresourceOperations would make more sense.

Reply:
done

Reply:
Nice, thank you!